### ISSUE TITLE
Facility Emissions Summary Page (LFO & SFO)

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-reporting#142]
Description: 

"As an LFO Industrial Reporter I want to see an emissions summary for each facility"
Note: SFO's will only see one emissions summary

We need to show Industry users a summary page for the emissions data for each facility. The fields on this page will not be editable and will be sums of emissions data entered in previous pages (activities). 

[Wireframe](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7826-18284&t=KLULsQuubY5EGgw7-4)

Acceptance Criteria: 
Given that I am an Industry Reporter
When I fill out the emissions report section of my annual report (i.e. all the activities forms)
Then I want to see a summary of emissions totals before moving on to the Production section of the report. 

Dev Checklist: 

**Note:** Once #333 is done, report_emission records should have 1 or more emission categories assigned to it

- [x] An API endpoint/service exists that aggregates report_emission records by emission category & returns the sums for each emission category
- [x] There is no "Save" on this page as it is just displaying information
- [x] The totals at the top of the page (attributable for reporting, threshold. reporting only & total facility) are calculated correctly based on the details in the excel sheet provided by IRC. File is in the (SME channel -> Reporting App -> Emissions Summary).
- [x] The page displays the totals by emission category as shown in the wireframe

Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ]  User story is included
- [ ]  User role and type are identified
- [x]  Acceptance criteria are included
- [ ]  Wireframes are included (if required)
- [ ]  Design / Solution is accepted by Product Owner
- [ ]  Dependencies are identified (technical, business, regulatory/policy)
- [ ]  Story has been estimated (under 13 pts)

Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ]  Acceptance criteria are tested by the CI pipeline
- [ ]  UI meets accessibility requirements
- [ ]  Configuration changes are documented, documentation and designs are updated
- [ ]  Passes code peer-review
- [ ]  Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ]  Ticket is ready to be merged to main branch
- [ ]  Can be demoed in Sprint Review
- [ ]  Bugs or future work cards are identified and created
- [ ]  Reviewed and approved by Product Owner

Blocked By

Notes:

**Comment by patriciarussellCAS** [comment]
Updated this ticket to reflect the need for an emissions summary page in the workflow. Tagging @suhafa for visibility.

**Comment by dleard** [comment]
@patriciarussellCAS @pbastia @suhafa blocked until we receive the "rules" about how to apply emission categories based on the data we collect. Adding the attn: Business Area tag

**Comment by rdromey** [comment]
@dleard to write dev checklist

**Comment by patriciarussellCAS** [comment]
@dleard new feedback from Jairo about this page (see [wireframe](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=17438-5877&t=JCeuChxKEFydADJw-4)). There are two fields he is suggesting we move from Emissions Summaries to the Compliance summary page. I will confirm and then if required, @zoeyli-46 we'll have to update the wireframes.

**Comment by zoeyli-46** [comment]
New feedback from Emilie "Emissions from fat, oil and grease collection, refining and storage isn't applicable to LFOs" @dleard

**Comment by patriciarussellCAS** [comment]
@dleard or @ayeshmcg - can you walk me through how I get to the emissions summary page through the workflow? 
I can get to the activity page, fill it out and hit submit and am not sent anywhere. 
If I can't navigate through the the flow, can you send me the direct link to the summary page for review? thanks,
